portal/templates/flask_user/emails/forgot_password_message.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'flask_user/emails/base_message.html' %}
+
+{% block message %}
+
+{% if app_name == "TrueNTH" %}
+
+<p>It sounds like you're having trouble logging in to TrueNTH.</p>
```

Comment:
My mistake- looks like [Flask-User maps USER_APP_NAME to app_name in templates](https://github.com/lingthio/Flask-User/blob/1e31e1b4c6a2766d9888a97b30398cdc96de1705/flask_user/settings.py#L21)